src/Components/CreateImageWizardV2/validators.ts

```diff
@@ -13,3 +13,22 @@ export const isGcpEmailValid = (gcpShareWithAccount: string | undefined) => {
     gcpShareWithAccount.length <= 253
   );
 };
+
+export const isBlueprintNameValid = (blueprintName: string | undefined) => {
+  if (blueprintName === '') {
+    return true;
+  }
+  return (
+    blueprintName !== undefined &&
+    /^[a-z][a-z0-9-]+[a-z0-9]$/.test(blueprintName) &&
+    blueprintName.length <= 63
+  );
+};
+
+export const isBlueprintDescriptionValid = (
+  blueprintDescription: string | undefined
+) => {
+  return (
+    blueprintDescription !== undefined && blueprintDescription.length <= 250
```

Comment:
We won't need to check for undefined here either if blueprintDescription is of type `string`.

Reply:
if I remove the blueprintDescription !== undefined I have an error:
TS18048: 'blueprintDescription' is possibly 'undefined'.